bestie/proto/BUILD.bazel

```diff
@@ -1,4 +1,5 @@
-load("@rules_python//python:proto.bzl", "py_proto_library")
+load("@com_google_protobuf//bazel:py_proto_library.bzl", "py_proto_library")
+load("@com_google_protobuf//bazel:cc_proto_library.bzl", "cc_proto_library")
```

Comment:
I can only assume this is the correct load rule for cc_proto_library. I don't have any other context for the changes being made in this PR.

Reply:
https://github.com/bazelbuild/rules_cc/blob/main/cc/defs.bzl#L49

Reply:
OK, thanks for the reference.